smart-agent-did-resolver-api/test/agent-tests.js

```diff
@@ -0,0 +1,60 @@
+const ActionHero = require('actionhero')
+const actionhero = new ActionHero.Process()
+const path = require('path')
+const chai = require('chai')
+const expect = chai.expect
+const request = require('request-promise-native')
+
+process.env.ACTIONHERO_CONFIG = path.join(__dirname, '../..', 'node_modules/@evan.network/edge-server-seed/config')
+process.env.PROJECT_ROOT = path.join(__dirname, '../..', 'node_modules/@evan.network/edge-server-seed/')
+process.env.NODE_ENV = 'test'
+
+describe('smart agent did resolver', function () {
+    this.timeout(60000)
+
+    let api;
+    let baseUrl = ""
+
+    before(async () => {
+        console.log(__dirname)
+      api = await actionhero.start({configChanges: {
+        plugins: {
+          'smart-agent-did-resolver': { path: `${__dirname}/..` }
+        }
+      }})
+      baseUrl = `http://localhost:${api.config.servers.web.port}/api`
+    })
+
+    after(
```

Comment:
Identitation looks a bit strange. Would suggest one of the following.

```ts
before(async () => {
  await actionhero.stop()
})
```

or

```ts
before(async () => { await actionhero.stop() })
```

Reply:
why actually the anon function? Isn't it possible to do `after async ( ...` ?